src/enkf/controlvec.f90

```diff
@@ -335,6 +337,9 @@ subroutine write_control(no_inflate_flag)
            else
               call writegriddata(0,0,cvars3d,cvars2d,nc3d,nc2d,clevels,ncdim,grdin_mean,no_inflate_flag)
            end if
+        elseif (nc2d>0) then  ! always write sfc mean increment for soil analysis
+           no_vars3d=''
```

Comment:
Just a clarifying question, @ClaraDraper-NOAA , so in operations, the ensemble mean increment file that is written out will be a full set of variables, this case where the 3D variables is skipped is only in certain situations? I ask because it seems like this would create a file with a bunch of variables that are all 0s which seems like a waste of disk space.